pkg/skaffold/errors/problem_catalog.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The Skaffold Authors
```

Comment:
nit:

```suggestion
Copyright 2021 The Skaffold Authors
```

Reply:
oops. will change in follow up. 
Another way to increase my commit count ;)